packages/xdl/src/Webpack.ts

```diff
@@ -363,3 +415,133 @@ async function getWebpackConfigEnvFromBundlingOptionsAsync(
     ...(options.webpackEnv || {}),
   };
 }
+
+async function startNextJsAsync({ projectRoot, port, dev, expoConfig, onFinished }) {
+  let next;
+  try {
+    const { exp } = await ProjectUtils.readConfigJsonAsync(projectRoot);
+    next = require(ConfigUtils.resolveModule('next', projectRoot, exp));
+  } catch {
+    throw new XDLError('NEXTJS_NOT_INSTALLED', 'Next.js is not installed in your app.');
```

Comment:
We should include instructions for fixing this or a link to the relevant page in the documentation.